src/main/webapp/WEB-INF/tags/module/ar/cashControlDetails.tag

```diff
@@ -0,0 +1,150 @@
+<%--
+
+    The Kuali Financial System, a comprehensive financial management system for higher education.
+
+    Copyright 2005-2023 Kuali, Inc.
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+--%>
+<%@ include file="/jsp/sys/kfsTldHeader.jsp"%>
+
+<SCRIPT type="text/javascript">
+    function hideImport() {
+        document.getElementById("showLink").style.display="inline";
+        document.getElementById("uploadDiv").style.display="none";
+    }
+    function showImport() {
+        document.getElementById("showLink").style.display="none";
+        document.getElementById("uploadDiv").style.display="inline";
+    }
+</SCRIPT>
+
+<!-- CU Customization: Backport FINP-10040 This file can be removed after upgrading to the 2023-09-20 financials patch. -->
+
+<%@ attribute name="documentAttributes" required="true"
+              type="java.util.Map"
+              description="The DataDictionary entry containing attributes for cash control document fields."%>
+<%@ attribute name="cashControlDetailAttributes" required="true"
+              type="java.util.Map"
+              description="The DataDictionary entry containing attributes for cash control detail fields."%>
+<%@ attribute name="readOnly" required="true"
+              description="If document is in read only mode"%>
+<%@ attribute name="editDetails" required="true"
+              description="If document details are in edit mode"%>
+<%@ attribute name="editPaymentAppDoc" required="true"
+              description="If payment application document number should be a link"%>
+
+<kul:tab tabTitle="Cash Control Details" defaultOpen="true" tabErrorKey="${KFSConstants.CASH_CONTROL_DETAILS_ERRORS}"
+         helpUrl="${KualiForm.detailsImportInstructionsUrl}" helpLabel="Import Templates">
```

Comment:
Can you confirm the help URL works as expected

Reply:
I think it is working now after I added a new parameter component definition